### ISSUE TITLE
Strip down readme and update website

### ISSUE THREAD

**Issue by jaredpalmer** [formium/tsdx#848]
### Current Behavior

I forgot to strip down the readme when doing the site. It's confusing to have docs on both the readme and website as there is no single source of truth. 
### Expected behavior

The website is the source of truth for documentation. 

### Suggested solution(s)

Strip down readme to logo, one-liner description, badges, links to website, plus contributors. Update website to remove any discrepancies w/ current readme.

**Comment by agilgur5** [comment]
I agree it's confusing if they're different (they are slightly right now), but if they're the same then 🤷  Could do the same thing as the Releases/Changelog page and just pull the README into the website.

I'm not sure I agree with the suggested solution. It's always annoying when you're on GitHub and you find some interesting repo but there's little to nothing in the README and it just points to a separate website. It requires another click which is generally not good UX (there's drop-off/that requires another conversion) and isn't great for SEO either

**Comment by agilgur5** [comment]
Per my last email, I'd also appreciate if the Vercel bot were turned off as it's giving me a massive amount of unnecessary notifications instead of using GitHub status checks and only running on website PRs/commits like Netlify did.

**Comment by jaredpalmer** [comment]
Gonna have to disagree with you about docs. As someone that has dealt with this before, Google figures out rather quickly when documentation is moved over. 

The readme should look like this: 

<img width="1151" alt="Screen Shot 2020-08-31 at 12 37 12 PM" src="https://user-images.githubusercontent.com/4060187/91743967-b307cf80-eb86-11ea-8289-626f3f167fcc.png">

**Comment by agilgur5** [comment]
> As someone that has dealt with this before, Google figures out rather quickly when documentation is moved over.

That's not what I said -- I'm saying that it de-prioritizes the actual repo since the content is elsewhere. Things like this can also happen:

here's `mobx-state-tree`'s results:

![Screen Shot 2020-08-31 at 12 42 39 PM](https://user-images.githubusercontent.com/4970083/91744519-a0da6100-eb87-11ea-8fd6-0c80ca8acf73.png)

vs. Jest's results:

![Screen Shot 2020-08-31 at 12 42 58 PM](https://user-images.githubusercontent.com/4970083/91744636-c5ced400-eb87-11ea-8e30-953d827537de.png)


Jest is a good example too because [it still has a README](https://github.com/facebook/jest). And each package has its own README that is not on the website too. I have well-documented plans to make TSDX itself into a monorepo to handle things like Presets (#634 ) and Shareable Plugins (#635 ), among other things. So these add questions of how that should be handled with a website (with READMEs, it's pretty straightforward).

**Comment by jaredpalmer** [comment]
Go for it then. Either way the website should be updated or symlinked such that contributors know where to make edits.
